package.json

```diff
@@ -83,6 +84,7 @@
     "eslint": "8.10.0",
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-local": "1.0.0",
+    "get-stream": "^6.0.1",
```

Comment:
Exact version

Reply:
Darn. I keep missing those, I need to write myself a git commit hook or something.